soh/src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -164,7 +164,7 @@ void BossGanon2_Init(Actor* thisx, PlayState* play) {
     }
 
     this->actor.colChkInfo.mass = MASS_IMMOVABLE;
-    this->actor.colChkInfo.health = 30;
+    this->actor.colChkInfo.health = 1;
```

Comment:
These changes were for debugging (fast kill ganon) need to remove
```suggestion
    this->actor.colChkInfo.health = 30;
```